### PR TITLE
Fixed typos and errors in the Roadmap.

### PR DESCRIPTION
**Pull request by Ameenafz248**
Descriptions are given where needed in each of the commits.

### CODE REVIEW

File: docs/Roadmap/Stage04.md [hunk 1]
```diff
@@ -518,5 +518,5 @@ SELECT * FROM Students INTO null WHERE Marks>=90;
 ```plain
 SELECT * FROM RELATIONCAT INTO null WHERE #Records > five;  # Error: Mismatch in attribute type
 SELECT * FROM RELATIONCAT INTO null WHERE Name = Students;  # Error: Attribute does not exist
-SELECT * FROM Students INTO null WHERE Batch=J;             # Error: Relation is not open
+SELECT * FROM Participants INTO null WHERE regNo>0;         # Error: Relation is not open
```
Comment (clifordjoshy) [inline docs/Roadmap/Stage04.md:521]: Why is this change needed again? Is it because the relation students will be open from the exercises in the previous question?

File: docs/Roadmap/Stage08.md [hunk 2]
```diff
@@ -321,7 +321,7 @@ Your NITCbase now supports the creation of relations. With that, we have now imp
 
 ## Exercises
 
-**Q1.** In your NITCbase, run the file [s8test.txt](/roadmap_files/stage8/script.txt) to test your implementation. Place the files [products.csv](/roadmap_files/stage8/products.txt) and [stores.csv](/roadmap_files/stage8/stores.txt) in the `Files/Input_Files` directory. Place [s8test.txt](/roadmap_files/stage8/script.txt) in the `Files/Batch_Execution_Files` directory. Once you have placed the files, execute the [run](../User%20Interface%20Commands/utility.md#run-batch-execution-command) command in your NITCbase as below.
+**Q1.** In your NITCbase, run the file [s8test.txt](/roadmap_files/stage8/script.txt) to test your implementation. Place the files [s8products.csv](/roadmap_files/stage8/products.txt) and [s8stores.csv](/roadmap_files/stage8/stores.txt) in the `Files/Input_Files` directory. Place [s8test.txt](/roadmap_files/stage8/script.txt) in the `Files/Batch_Execution_Files` directory. Once you have placed the files, execute the [run](../User%20Interface%20Commands/utility.md#run-batch-execution-command) command in your NITCbase as below.
```
Comment (clifordjoshy) [inline docs/Roadmap/Stage08.md:324]: What is the rationale for this naming?

**Comment by clifordjoshy** [comment]
Thanks for catching all these issues. Looks great! Just had a few queries. Will merge it once those are resolved.

File: docs/Roadmap/Stage04.md (hunk 1 above)
Comment (clifordjoshy) [reply docs/Roadmap/Stage04.md:521]: Just noticed your comment on the commit. The student relation is not meant to be left hardcoded forever. I suppose this serves as a check to remove that if you have not done so at the end of the stage. In the next stage, relation opening will be implemented anyway, right? Let me know if I'm misunderstanding something.

**Review by clifordjoshy — CHANGES_REQUESTED** [review]

**Comment by clifordjoshy** [comment]
Also, wherever you've made changes to the design, could you make the same changes on the [stub files](https://nitcbase.github.io/docs/Misc/stub/frontend)?

File: docs/Roadmap/Stage04.md (hunk 1 above)
Comment (Ameenafz248) [reply docs/Roadmap/Stage04.md:521]: But if the student follow the tutorial as such, the error mentioned, 'Error: Relation is not open' will not print as it is already opened.

File: docs/Roadmap/Stage08.md (hunk 2 above)
Comment (Ameenafz248) [reply docs/Roadmap/Stage08.md:324]: The file give to run uses the name s8stores.txt s8products.txt for inserting records. Here is that section from that code: 

echo ----Populating new relations----
INSERT INTO Products VALUES FROM s8products.csv
INSERT INTO Stores VALUES FROM s8stores.csv
echo --------------------------------
echo
echo ----Selecting from new relation----
SELECT * FROM Stores INTO null WHERE owner=Gomez
echo -----------------------------------

It might be better if we change the names in the documentation or in the command file s8.txt

File: docs/Roadmap/Stage04.md (hunk 1 above)
Comment (clifordjoshy) [reply docs/Roadmap/Stage04.md:521]: Makes sense

**Review by clifordjoshy — APPROVED** [review]
LGTM. Thanks again!
